tests/containers/rootless_podman.pm

```diff
@@ -22,13 +22,28 @@ use containers::common;
 use containers::container_images;
 use containers::urls 'get_suse_container_urls';
 use containers::utils 'registry_url';
-use version_utils 'is_sle';
+use version_utils qw(is_sle is_leap);
+use power_action_utils 'power_action';
+use bootloader_setup 'add_grub_cmdline_settings';
 use Utils::Architectures;
 
 sub run {
     my ($self) = @_;
     $self->select_serial_terminal;
 
+    if (is_sle('15-SP4+') || is_leap('15.4+')) {
```

Comment:
```suggestion
    if (is_sle('15-SP3+') || is_leap('15.4+')) {
```

15-SP3 is probably going to release an update to `3.4.4` where this issue happens and we need to workaround it as well. I'm not sure about 15.3 tough.

Reply:
@Vogtinator what should we do about 15.3? Is there a plan to update podman to 3.4.4 as well there?

Reply:
I did not succeed with CGroups v2 on Leap 15.4.